### PR TITLE
Feature/sw 24

### PR DESCRIPTION
**Pull request by dangvu2b**

### CODE REVIEW

File: modules/api/src/models/collection.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import mongoose from '@/infrastructure/mongoose'
+
+const schema = new mongoose.Schema({
+  id: {
+    type: String,
+    required: true
+  },
+  title: {
+    type: Number
```
Comment (truongcuong90) [inline modules/api/src/models/collection.js]: Change  "title" type "Numeber" to String

File: modules/api/src/models/collection.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  productsCount: {
+    type: Number
+  },
+  image: {
+    type: mongoose.Schema.Types.Mixed,
+    default: {}
+  },
```
Comment (truongcuong90) [inline modules/api/src/models/collection.js]: Remove Field Image  if it not use

File: documentations/get-products-by-collections-sq.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+boundary Client(React)
+control Shopify App
+control API
+materialdesignicons FADD Shopify
+
+Client(React)->Shopify: Resource picker by Collection
+Shopify-->Client(React): Collections
+loop collectionId in collectionIds
+Client(React)->Shopify App: Request get products
+Shopify App->API:GET /collections/c_id/products
```
Comment (truongcuong90) [inline documentations/get-products-by-collections-sq.txt]: Rename "c_id" to collectionId

File: modules/api/src/resources/shopify-store/collection/product/index.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            const { token: accessToken, identifier } = shopifyStore
+            const requestProps = {
+              method: 'GET',
+              headers: {
+                'X-Shopify-Access-Token': accessToken,
+                'Content-Type': 'application/json'
+              }
+            }
+
+            const response = await fetch(`https://${identifier}/admin/api/2020-01/collections/${collectionId}/products.json?limit=5&page_info=`,
```
Comment (longlh) [inline modules/api/src/resources/shopify-store/collection/product/index.js]: Calling to Shopify API should have a loop for retrying if request limitation reaches

Comment (namnq) [inline modules/api/src/resources/shopify-store/collection/product/index.js]: make number limit 5 to const or something can configure

**Pull request merged**